hardhat.config.ts

```diff
@@ -62,14 +75,21 @@ const config: HardhatUserConfig = {
         "node_modules/@keep-network/tbtc/artifacts",
         "./external/ropsten",
       ],
+      mainnet: ["./external/mainnet"],
     },
   },
   namedAccounts: {
     deployer: {
       default: 0, // take the first account as deployer
+      ropsten: RopstenSecrets.address,
+      mainnet: MainnetSecrets.address,
     },
     rewardManager: {
       default: 1,
+      mainnet: "0xB3726E69Da808A689F2607939a2D9E958724FC2A", // Technical Wallet // TODO: Technical wallet or rewards manager?
```

Comment:
@pdyraga would you please double-check this https://github.com/keep-network/coverage-pools/issues/125#issuecomment-882472963.

Reply:
Let's use deployer address, this role is later transferred anyway.

Reply:
This still has to be reworked, right?

Reply:
Done now